README.md

```diff
@@ -1,5 +1,12 @@
 # lantern-front-end
 
+## Configuration
+This project reads the following environment variable:
+
+* **GRAFANA_URL**: The grafana dashboard url that is being used for displaying visualizations.
```

Comment:
i think this should include an application-based lead-in, like, "LANTERN_GRAFANA_URL". Obviously, "lantern" is subject to change, but you could call it that for now so when we do the find/replace, it will be caught.

Reply:
Makes sense to me!